cloud/google/machineactuator.go

```diff
@@ -201,22 +198,18 @@ func (gce *GCEClient) Create(cluster *clusterv1.Cluster, machine *clusterv1.Mach
 		return errors.New("invalid master configuration: missing Machine.Spec.Versions.Kubelet")
 	}
 
-	machineSetupConfigs, err := gce.configWatch.ValidConfigs()
-	if err != nil {
-		return err
-	}
 	configParams := &machinesetup.ConfigParams{
 		OS:       config.OS,
 		Roles:    machine.Spec.Roles,
 		Versions: machine.Spec.Versions,
 	}
-	image, err := machineSetupConfigs.GetImage(configParams)
+	image, err := gce.machineSetupConfig.GetImage(configParams)
 	if err != nil {
 		return err
 	}
 	imagePath := gce.getImagePath(image)
 
-	machineSetupMetadata, err := machineSetupConfigs.GetMetadata(configParams)
+	machineSetupMetadata, err := gce.machineSetupConfig.GetMetadata(configParams)
```

Comment:
What guarantees do we have that the configuration has not updated between the time that we retrieved the image, and when we retrieved the metadata? Before there was a guarantee that we were operating on the same data for this create.